packages/teams-js/src/public/dialog.ts

```diff
@@ -17,21 +17,30 @@ import { runtime } from './runtime';
  *
  * @beta
  */
-
-export interface SdkResponse {
-  err?: string;
-  result?: string | object;
-}
-
 export namespace dialog {
+  /**
+   * Data Structure to represent the SDK response when dialog closes
+   */
+  export interface ISdkResponse {
+    /**
+     * Error in case there is a failure before dialog submission
+     */
+    err?: string;
+
+    /**
+     * Result value that the dialog is submitted with
+     */
```

Comment:
I believe this comes from dialog.submit function right? If so, let's call attention to that.

Reply:
(There's a way to make a link to another function in the documentation, @link maybe?)